proxy/proxy/rhnShared.py

```diff
@@ -151,7 +151,7 @@ def _connectToParent(self):
             log_error("Error opening connection", self.rhnParent, e)
             Traceback(mail=0)
             raise_with_tb(rhnFault(1000,
-                                   _("SUSE Manager Proxy could not successfully connect its SUSE Manager parent. "
+                                   _("SUSE Multi-Linux Manager Proxy could not successfully connect its SUSE Multi-Linux Manager parent. "
```

Comment:
you are back again to use the hardcoded name while in  `proxy/proxy/rhnProxyAuth.py` you defined `PRODUCT_NAME = "SUSE Multi-Linux Manager"` and you used  it.

it's a bit complex to review this PR given that you are using so many different ways to rebrand the product name. in some cases you define the variable with the string of the product name, in others you just hardcode the name. TBH, here I don't get if this is a leftover of a previous attempt to rebrand  or if there's a specific reason for hardcoding the product name...

Reply:
There are various types of files involved, each requiring a bit different approach, so there isn't a one-size-fits-all solution, in this case. Mostly focus has been on renaming instead of refactoring. I will make this change, but we should proceed with merging this PR. Since a lot of translation work is involved, we should merge it soon so we have enough time fix the issues. Any remaining issues can be addressed in a separate PR.